tests/explainers/test_gradient.py

```diff
@@ -97,12 +97,12 @@ def test_tf_keras_mnist_cnn(random_seed):
     e = shap.GradientExplainer((model.layers[0].input, model.layers[-1].input), x_train[inds, :, :])
     shap_values = e.shap_values(x_test[:1], nsamples=2000)
 
-    diff = sess.run(model.layers[-1].input, feed_dict={model.layers[0].input: x_test[:1]}) - \
-    sess.run(model.layers[-1].input, feed_dict={model.layers[0].input: x_train[inds, :, :]}).mean(0)
+    outputs = sess.run(model.layers[-1].input, feed_dict={model.layers[0].input: x_test[:1]})
+    background = sess.run(model.layers[-1].input, feed_dict={model.layers[0].input: x_train[inds, :, :]})
+    expected_value = background.mean(0)
 
-    sums = np.array([shap_values[i].sum() for i in range(len(shap_values))])
-    d = np.abs(sums - diff).sum()
-    assert d / (np.abs(diff).sum() + 0.01) < 0.1, "Sum of SHAP values does not match difference! %f" % (d / np.abs(diff).sum())
+    sums = shap_values.sum((1, 2, 3))
+    assert np.allclose(sums + expected_value, outputs, atol=1e-4)
```

Comment:
This new syntax for the assert is a lot clearer, nice. As a minor nitpick, `np.testing.assert_allclose` might be slightly better than `assert np.allclose`. It's the same function, but with slightly more appropriate defaults and more informative error tracebacks.